plugins/rebalance/rest.go

```diff
@@ -0,0 +1,231 @@
+package rebalance
+
+import (
+	//"context"
+	//"encoding/json"
+	"fmt"
+
+	"github.com/gluster/glusterd2/glusterd2/gdctx"
+	restutils "github.com/gluster/glusterd2/glusterd2/servers/rest/utils"
+	"github.com/gluster/glusterd2/glusterd2/store"
+	"github.com/gluster/glusterd2/glusterd2/transaction"
+	"github.com/gluster/glusterd2/glusterd2/volume"
+	"github.com/gluster/glusterd2/pkg/api"
+	"github.com/gorilla/mux"
+	"github.com/pborman/uuid"
+	log "github.com/sirupsen/logrus"
+	"net/http"
+)
+
+type Rebelstatus uint64
```

Comment:
s/Rebel/Rebal/

Its rebalance not rebelence. It would be nicer to just call this `RebalanceStatus` instead of shortening.